src/runtime.js

```diff
@@ -49,6 +50,17 @@ const TAG = 'runtime';
 /** @type {!Array} */
 const elementsForTesting = [];
 
+// TODO
+if ('serviceWorker' in navigator) {
+  const url = `${calculateScriptBase(window)}/v0_sw.max.js`;
```

Comment:
It doesn't even work cross origin for now. The "foreign fetch" extension (unimplemented) would make this work.